scripts/base/protocols/ssh/main.zeek

```diff
@@ -53,7 +53,9 @@ export {
 		## The server host key's algorithm
 		host_key_alg:    string       &log &optional;
 		## The server's key fingerprint
-		host_key:        string       &log &optional;
+		host_key:        string       &log &optional &deprecated="Remove in 9.1. Use host_key_fingerprint.";
+		## The server's key fingerprint, in the format that `ssh-keygen -l` would output.
```

Comment:
Maybe add an example? I'm forth and back if the  `SSH256:<base64>` format is nice, but I think it works.